packages/chai-dom-equals/test/chai-dom-equals.test.js

```diff
@@ -152,19 +152,19 @@ describe('getCleanedShadowDom', () => {
 });
 
 describe('dom', () => {
-  it('can strictly compare dom nodes', async () => {
+  it('can compare dom nodes', async () => {
     const el = await fixture(`<div><!-- comment --><h1>${'Hey'}  </h1>  </div>`);
-    expect(el).dom.to.equal('<div><!-- comment --><h1>Hey  </h1>  </div>');
```

Comment:
so if I want to do a "strict" comparison of dom I would need to write it myself right?
e.g.
```
expect(el.innerHTML).to.equal('<div><!-- comment --><h1>Hey  </h1>  </div>');
```

problem is that this will also include all the shadycss markers if you use shadodom... so is there a solution to compare strict but to ignore the shadycss stuff?

Reply:
Updated readme